src/main/java/crawlercommons/sitemaps/sax/XMLIndexHandler.java

```diff
@@ -72,30 +74,24 @@ public void endElement(String uri, String localName, String qName) throws SAXExc
             maybeAddSiteMap();
         } else if ("sitemapindex".equals(currentElement())) {
             sitemap.setProcessed(true);
+        } else if ("loc".equals(currentElement())) {
+            locFound = true;
```

Comment:
Would we want to set `locFound = false` in the `endElement()` method, for we get `</loc>`? Maybe it doesn't matter...